src/Makefile

```diff
@@ -458,6 +458,7 @@ $($(2)_DOTNET_BUILD_DIR)/$(4)/Microsoft.$(1)%dll $($(2)_DOTNET_BUILD_DIR)/$(4)/M
 	$$(call Q_PROF_CSC,dotnet/$(4)-bit) \
 		$(DOTNET_CSC)  \
 		$(DOTNET_FLAGS) \
+		/analyzer:$(ROSLYN_GENERATOR_COMMON) \
```

Comment:
This is a very annoying situation with roslyn code generators, we need to put the dependency .csproj as a code generator for it to work, else we get a FileNotFound exception when we try to load the dll from the code generator.